pcsx2/DEV9/ConfigUI.cpp

```diff
@@ -184,16 +184,15 @@ class DEV9Dialog : public wxDialog
 		hdd_grid->Add(m_hdd_file,     wxSizerFlags().Expand());
 		auto* hdd_size_label = new wxStaticText(this, wxID_ANY, _("HDD Size (GiB):"));
 		auto* hdd_size_box = new wxBoxSizer(wxHORIZONTAL);
-		m_hdd_size_slider = new wxSlider(this, wxID_ANY, HDD_MIN_GB, HDD_MIN_GB, HDD_MAX_GB, wxDefaultPosition, wxDefaultSize, wxSL_HORIZONTAL | wxSL_MIN_MAX_LABELS | wxSL_AUTOTICKS);
+		m_hdd_size_slider = new wxSlider(this, wxID_ANY, 10, HDD_MIN_GB, HDD_MAX_GB, wxDefaultPosition, wxDefaultSize, wxSL_HORIZONTAL | wxSL_AUTOTICKS);
```

Comment:
The value you replaced with `10` is the initial value of the slider.  Putting a value less than the minimum just ends up with the minimum in a more confusing way.

Reply:
Forgot I changed the value. Thanks.